webapp/app/sessions/template.hbs

```diff
@@ -5,11 +5,16 @@
   {{#if (not hide_search)}}
     <form class="navbar-form" role="search">
       <div class="input-group col-xs-6 {{if error 'has-error'}}">
-        {{input class="form-control search-bar" placeholder="Search for..." value=entered_search}}
+        {{input class="form-control search-bar" id="search-field" placeholder="Search for..." value=entered_search}}
       </div>
       <div class="input-group">
         <button class="btn btn-default" type="submit" {{action "search"}}><i class="fa fa-search"></i></button>
       </div>
+      <script>
+        $('#search-field').on('keyup', function(e) {
```

Comment:
This does not guarantee the DOM is ready when the script is executed... It is better encapsulated as a component for the search bar...